core/src/main/resources/lib/ophan.fjs

```diff
@@ -48,5 +49,8 @@ export const Actions: { [Action]: Action } =
   , ANSWER: 'ANSWER'
   , VOTE: 'VOTE'
   , CLICK: 'CLICK'
+  , PLAY: 'PLAY'
```

Comment:
These will have to be present in the Ophan Thrift definition too.

Reply:
This is already defined here: https://github.com/guardian/ophan/blob/master/event-model/src/main/thrift/media.thrift#L25 do I need to add it again somewhere else?

Reply:
Hmm, then you will need to walk your way up in the Thrift definition to find where this stuff is used. Let me have a look. So in order:
- `MediaPlayback`: requires the media id, do we have that?
- `Event`: this is the payload generated by Ophan when you hit `ophan.record`.

So I think this is what you should do:
1- remove the changes to `Action`
2- add a new type, `MediaEvent` that is the copy of the Thrift definition: replace `Action` by `MediaEvent` everywhere it makes sense
3- update `record` to either send a `componentEvent` (for the `VIEW` action), or a `media` (for `PLAY`, `READY`, etc.). The shape of the latter should be something like this:
```js
ophan.record({
  media: {
    mediaId: xxx,
    mediaType: 'AUDIO',
    preroll: false,
    eventType: 'READY'
  }
})
```